### PR TITLE
add convertermanagement, consistent renaming of sql(2|to)regex

### PR DESCRIPTION
**Pull request by binkertpat**

### CODE REVIEW

File: src/main/java/sqltoregex/ConverterManagement.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+    /**
+     * compared list elements about unique elements
+     * @param list List<String>
+     * @return boolean
+     */
+    private Boolean checkOfDistinctElements(List<String> list){
+        if(list.isEmpty()) throw new IllegalArgumentException("List should not be empty. One element is required.");
+        Set<String> set = new HashSet<>(list);
+        return (set.size() == list.size());
```
Comment (Alletkla) [inline src/main/java/sqltoregex/ConverterManagement.java:68]: Problem might be, that List contains 1 duplicate and 2 same Strings.
Then set.size() is 1 less then list.size() but still has duplicates.

File: src/main/java/sqltoregex/deparser/SelectDeparser.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package sqltoregex.deparser;
+
+import net.sf.jsqlparser.util.deparser.SelectDeParser;
+
+public class SelectDeparser extends SelectDeParser {
+    public SelectDeparser() {
+        super();
```
Comment (Alletkla) [inline src/main/java/sqltoregex/deparser/SelectDeparser.java:7]: Dev time solution? Will an own implementation follow?

**Review by Alletkla — COMMENTED** [review]
lets talk about it

File: src/main/java/sqltoregex/deparser/StatementDeparser.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    public StatementDeparser(StringBuilder buffer) {
+        super(buffer);
+    }
+
+    public StatementDeparser(ExpressionDeParser expressionDeParser, SelectDeParser selectDeParser, StringBuilder buffer) {
+        super(expressionDeParser, selectDeParser, buffer);
+    }
+
+    public StatementDeparser(ExpressionDeParser expressionDeParser, StringBuilder buffer) {
+        super(expressionDeParser, new SelectDeparser(), buffer);
```
[the remark this answers is not in the document]
Comment (binkertpat) [reply src/main/java/sqltoregex/deparser/StatementDeparser.java:17]: DeParser vs. Deparser, import package as ...

File: src/main/java/sqltoregex/ConverterManagement.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (binkertpat) [reply src/main/java/sqltoregex/ConverterManagement.java:68]: isDistinctList

File: src/main/java/sqltoregex/deparser/SelectDeparser.java (hunk 2 above)
Comment (binkertpat) [reply src/main/java/sqltoregex/deparser/SelectDeparser.java:7]: yes! yes!

File: src/test/java/sqltoregex/ConverterManagementTest.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+import net.sf.jsqlparser.JSQLParserException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.web.servlet.MockMvc;
+
+@SpringBootTest
```
[the remark this answers is not in the document]
Comment (binkertpat) [reply src/test/java/sqltoregex/ConverterManagementTest.java:11]: didnt know this

File: src/test/java/sqltoregex/ConverterManagementTest.java [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+@AutoConfigureMockMvc
+class ConverterManagementTest {
+    @Autowired
+    MockMvc mvc;
```
[the remark this answers is not in the document]
Comment (binkertpat) [reply src/test/java/sqltoregex/ConverterManagementTest.java:15]: youre right

File: src/test/java/sqltoregex/PropertyTest/OrderRotationTest.java [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import sqltoregex.property.OrderRotation;
+
+import java.util.Arrays;
+import java.util.List;
+
+@SpringBootTest
```
[the remark this answers is not in the document]
Comment (binkertpat) [reply src/test/java/sqltoregex/PropertyTest/OrderRotationTest.java:11]: yeah yeah

File: src/test/java/sqltoregex/PropertyTest/SpellingMistakeTest.java [hunk 7]
```diff
@@ -0,0 +1,24 @@
+package sqltoregex.PropertyTest;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import sqltoregex.property.SpellingMistake;
+
+@SpringBootTest
```
[the remark this answers is not in the document]
Comment (binkertpat) [reply src/test/java/sqltoregex/PropertyTest/SpellingMistakeTest.java:8]: yeah yeah

**Review by Alletkla — APPROVED** [review]

**Pull request merged**